content/analyst/inbox-and-design-tracker/inbox-tracker-reference-guide.md

```diff
@@ -0,0 +1,95 @@
+* **Begin seeding.** In addition to our panel, you can send to a seed list to get better deliverability coverage across your campaigns. Refer to our [seeding guide(s)](/analyst/inbox-and-design-tracker/inbox-design-tracker#seeding) for guidance on integrating these addresses into your mail streams to increase the amount of campaign-level data.
+* **Setting up alerts.** If you want to monitor your deliverability, traps, blocklists, etc. from afar, alerts are a great way to do that. Here is [a guide](/analyst/inbox-and-design-tracker/getting-started-with-alerts) on how to create a base-set of alerts to begin monitoring changes in your set up or performance.
+
+![](media/inbox_tracker_reference_guide/image_2.png)
+
+*Alerts can be configured to go through your email or via [Slack](/analyst/inbox-and-design-tracker/how-to-configure-slack-for-inbox-tracker)*
+
+---
+
+### General Platform Navigation and Daily, Weekly and Monthly Goals
+
+**Daily/Weekly Goals** 
+
+* Review Spam Complaints, Spam Trap Hits, Blocklists and identify problematic campaigns that need further review.
+* If your campaigns are time sensitive, review Average Time To Receive in Campaign Tracker to identify any delays from ISP throttling.
+* Test variations of each Subject Line in Subject Line Predict prior to deployment to optimize open rates and compare against your industry average. Subscriber engagement impacts deliverability.
+* Check your delivery alerts and Reputation Advisor report to stay on top of your deliverability trends.
+
+**Monthly Goals** 
+
+* Monitor your Inactive Rate. The goal is 40% or below inactive. If your inactive rate is above 40%, consider your engagement based segmentation for domains with low inbox placement, a programmatic reactivation series and identify a point at which you will no longer mail an inactive subscriber.
+* Periodically include the **[Analytics Tracking Pixel](/analyst/inbox-and-design-tracker/how-to-use-analytics-tracker)** in a campaign to learn more about your subscribers including how much time they spend reading your email, whether they open on desktop or mobile, their preferred device and browser.
+* Review **[Subject Line Advisor’s](/analyst/inbox-and-design-tracker/analyze-and-optimize-subject-line-performance)** sentiment analysis to determine which emotions and subject lines are resonating with your audience.
+
+---
+
+### Understanding your Results
+
+
+* **[Cloudmark Troubleshooting Guide](/analyst/inbox-and-design-tracker/cloudmark-reputation-troubleshooting-guide).** Learn more about Cloudmark and get direction on how to handle email content flags.
+* **[How to View/Interpret Your Google Postmaster Data](/analyst/inbox-and-design-tracker/how-to-view-interpret-your-google-postmaster-data)**
```

Comment:
Link broke on intercom page and here. I assume this is supposed to point to `/analyst/inbox-and-design-tracker/how-to-interpret-google-postmaster-tools/`